schema/spacewalk/upgrade/susemanager-schema-4.4.3-to-susemanager-schema-4.4.4/000-uyuni-fix-ol9-repos.sql

```diff
@@ -0,0 +1,36 @@
+INSERT INTO rhnContentSource (id, org_id, type_id, source_url, label, metadata_signed)
+  SELECT sequence_nextval('rhn_chan_content_src_id_seq'), CS.org_id, CS.type_id,
+         'https://yum.oracle.com/repo/OracleLinux/OL9/baseos/latest/x86_64/', 'Extern - Oracle Linux 9 (x86_64)', 'N'
```

Comment:
```suggestion
         'https://yum.oracle.com/repo/OracleLinux/OL9/baseos/latest/x86_64/', 'External - Oracle Linux 9 (x86_64)', 'N'
```

![image](https://user-images.githubusercontent.com/4226070/208689413-c3b6f640-97ac-4d80-9a11-216265b3eecb.png)

Reply:
No, this does not work. We need a different name. There is a unique index on it